scripts/updateUsers.ts

```
@@ -38,12 +42,367 @@ interface UserData {
+
+    try {
+      const result = await db.transaction(async (manager: any) => {
+        return processSingleBatch(manager, batchAddresses, userDataMap);
+      });
+
+      totals.users += result.users;
+      totals.workers += result.workers;
+      totals.userStats += result.userStats;
+      totals.workerStats += result.workerStats;
+
+      console.log(`  Batch complete: ${result.users} users | ${result.workers} workers | ${result.userStats} userStats | ${result.workerStats} workerStats`);
```

ESLint

[error] Replace ``··Batch·complete:·${result.users}·users·|·${result.workers}·workers·|·${result.userStats}·userStats·|·${result.workerStats}·workerStats`` with `⏎········`··Batch·complete:·${result.users}·users·|·${result.workers}·workers·|·${result.userStats}·userStats·|·${result.workerStats}·workerStats`⏎······`

(prettier/prettier)